OrbitCore/LinuxUtils.cpp

```diff
@@ -240,4 +243,40 @@ bool IsKernelOlderThan(const char* a_Version) {
   return GetKernelVersion() < GetVersion(a_Version);
 }
 
+//-----------------------------------------------------------------------------
+std::string GetProcessDir(pid_t process_id) {
+  return "/proc/" + std::to_string(process_id) + "/";
+}
+
+//-----------------------------------------------------------------------------
+std::map<uint32_t, std::string> GetThreadNames(pid_t process_id) {
+  std::string threads_dir = GetProcessDir(process_id) + "task/";
+  std::map<uint32_t, std::string> thread_ids_to_name;
```

Comment:
I can't because we're serializing the map through "cereal" and it doesn't know about flat_hash_map.  I could use a std::unordered_map, but figured it's cleaner to have the thread Ids sorted.

Reply:
Ah, got it. I think it might be good to add a comment explaining the choice of std::map here.